modules/encoders/cmd/brace.rb

```diff
@@ -27,7 +27,7 @@ def encode_block(state, buf)
     return buf if state.badchars !~ /\s/
 
     # Perform brace expansion encoding
-    "{#{buf.gsub(',', '\\,').gsub(/\s+/, ',')}}"
+    "{#{buf.gsub(/([{,}])/, '\\\\\1').gsub(/\s+/, ',')}}"
```

Comment:
This is terrible.

Reply:
But it works 👍

Reply:
so many backslashes i don't even know what's real anymore